docs/modules/servers/pages/distributed/operate/logging.adoc

```diff
@@ -38,72 +40,187 @@ link:https://www.elastic.co/elasticsearch[ElasticSearch].
+                        <timestampFormat>yyyy-MM-dd'T'HH:mm:ss.SSSX</timestampFormat>
+                        <timestampFormatTimezoneId>Etc/UTC</timestampFormatTimezoneId>
+
+                        <!-- Importance for handling multiple lines log -->
+                        <appendLineSeparator>true</appendLineSeparator>
+
+                        <jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter">
+                            <prettyPrint>false</prettyPrint>
+                        </jsonFormatter>
+                    </layout>
                 </encoder>
         </appender>
 
-        <appender name="ELASTIC" class="com.linagora.logback.elasticsearch.ElasticsearchAppender">
-            <url>http://elasticsearch:9200/_bulk</url>
-            <index>logs-james-%date{yyyy.MM.dd}</index>
-            <type>tester</type>
-            <includeMdc>true</includeMdc>
-            <excludedMdcKeys>host</excludedMdcKeys>
-            <errorLoggerName>es-error-logger</errorLoggerName>
-            <properties>
-                <property>
-                    <name>host</name>
-                    <value>${HOSTNAME}</value>
-                    <allowEmpty>false</allowEmpty>
-                </property>
-                <property>
-                    <name>severity</name>
-                    <value>%level</value>
-                </property>
-                <property>
-                    <name>thread</name>
-                    <value>%thread</value>
-                </property>
-                <property>
-                    <name>stacktrace</name>
-                    <value>%ex</value>
-                </property>
-                <property>
-                    <name>logger</name>
-                    <value>%logger</value>
-                </property>
-            </properties>
-            <headers>
-                <header>
-                    <name>Content-Type</name>
-                    <value>application/json</value>
-                </header>
-            </headers>
-        </appender>
+        <root level="INFO"><appender-ref ref="LOG_FILE" />
+        </root>
 
-        <root level="WARN">
-                <appender-ref ref="ELASTIC" />
+        <root level="DEBUG"><appender-ref ref="LOG_FILE" />
         </root>
 
-        <logger name="es-error-logger" level="DEBUG" additivity="false">
-            <appender-ref ref="CONSOLE" />
-        </logger>
+        <root level="WARN">
+                <appender-ref ref="LOG_FILE" />
+        </root>
 
         <logger name="org.apache.james" level="INFO" />
+        <logger name="com.datastax.driver.core.QueryLogger.SLOW" level="DEBUG" />
 
 </configuration>
 ....
+
+First you need to create a logs folder to mount the `logs` folder of James outside container,
+then mount that folder to FluentBit. In case you need to use `sh command` in fluent-bit container please use image: `fluent/fluent-bit:1.5.7-debug`
```

Comment:
First you need to create a `logs` folder, then mount it to James container and to FluentBit.